action.yml

```diff
@@ -1,57 +1,44 @@
-name: "Upstream release"
-description: "Perform an upstream GitHub release"
+name: "Release bot"
+description: "Create a tag and push it"
 
 inputs:
   token:
-    description: "A GitHub token for creating a release"
+    description: "A GitHub token for creating a tag"
     required: true
-  slack_webhook_url:
-    description: "A Slack incoming Webhook URL"
+  component:
```

Comment:
Cannot we extract this somehow from action's context? Might be way more difficult than it sounds though.

Reply:
https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

`github.repository` maybe?

Reply:
Yes, thanks. I've thought about this before and even looked in the docs, but couldn't find it. I'll try to test if this is "it".